config/locales/stamp_duty.cy.yml

```diff
@@ -26,6 +24,21 @@ cy:
       option_isFTB: prynu am y tro cyntaf
       option_isSecondHome: prynu eiddo atodol neu eiddo prynu-i-osod
       tooltip_html: Codir Treth Stamp ar wahanol gyfraddau yn ddibynnol ar os ydych yn prynu am y tro cyntaf, yn prynu eich cartref nesaf neu'n prynu eiddo atodol neu eiddo prynu-i-osod. <br /><br />Yn gyffredinol, rydych yn cael eich dosbarthu yn brynwr tro cyntaf os ydych chi'n prynu eich unig neu brif gartref ac nad ydych erioed wedi bod yn berchen ar eiddo yn y Deyrnas Unedig na thramor. <a href="https://www.moneyadviceservice.org.uk/cy/articles/awgrymiadau-ar-arian-i-brynwyr-tro-cyntaf" target="_blank">Darllenwch fwy yn ein canllaw i brynwyr tro cyntaf</a>.
+    elsewhere:
+      title: Prynu yn yr Alban neu Cymru?
+      body: Os ydych chi'n prynu yn unrhyw un o'r gwledydd hyn, yna defnyddiwch y gyfrifiannell briodol i gyfrifo faint fyddwch chi'n ei dalu
+      england_ni:
+        title: "Cyfrifwch Dreth Stamp yng Nghymru, Lloegr neu Ogledd Iwerddon"
+        body: "Rhaid talu Treth Stamp ar eiddo yn Lloegr a Gogledd Iwerddon o hyd"
+        link: "/cy/tools/prynu-ty/cyfrifiannell-treth-stamp"
+      wales:
+        title: Cyfrifwch Dreth Trafodiadau Tir ar gyfer Cymru
+        body: Mae Treth Trafodiadau Tir ac (LTT) wedi disodli Treth Stamp yng Nghymru.
+        link: /cy/tools/prynu-ty/land-transaction-tax-calculator-wales
+      scotland:
+        title: Cyfrifwch Dreth Trafodiadau Tir ac Adeiladau ar gyfer yr Alban
+        body: Mae Treth Trafodiadau Tir ac Adeiladau (LBTT) wedi disodli Treth Stamp yn yr Alban.
+        link: /cy/tools/prynu-ty/land-and-buildings-transaction-tax-calculator-scotland
```

Comment:
Can you change to use the links in welsh?
scotland:
   link: "/cy/tools/prynu-ty/cyfrifiannell-treth-stamp"

wales:
- "/cy/tools/prynu-ty/cyfrifiannell-treth-trafodiadau-tir-cymru"

Reply:
@tomas-stefano OK, I have changed those. (N.B. "cyfrifiannell-treth-stamp" is for en/ni, "cyfrifiannell-treth-trafodion-tir-ac-adeiladau-alban" for scotland)